driver-sync/src/main/com/mongodb/client/MongoCluster.java

```diff
@@ -352,4 +360,115 @@ public interface MongoCluster {
      * @mongodb.driver.dochub core/changestreams Change Streams
      */
     <TResult> ChangeStreamIterable<TResult> watch(ClientSession clientSession, List<? extends Bson> pipeline, Class<TResult> resultClass);
+
+    /**
+     * Executes a client-level bulk write operation.
+     * This method is functionally equivalent to {@link #bulkWrite(List, ClientBulkWriteOptions)}
+     * with the {@linkplain ClientBulkWriteOptions#clientBulkWriteOptions() default options}.
+     * <p>
+     * This operation supports {@linkplain MongoClientSettings#getRetryWrites() retryable writes}.
+     * Depending on the number of {@code models}, encoded size of {@code models}, and the size limits in effect,
+     * executing this operation may require multiple {@code bulkWrite} commands.
+     * The eligibility for retries is determined per each {@code bulkWrite} command:
+     * {@link ClientUpdateManyModel}, {@link ClientDeleteManyModel} in a command render it non-retryable.</p>
+     * <p>
+     * This operation is not supported by MongoDB Atlas Serverless instances.</p>
```

Comment:
See https://github.com/mongodb/specifications/blob/master/source/crud/bulk-write.md#is-bulkwrite-supported-on-atlas-serverless.